composer.json

```diff
@@ -77,6 +78,7 @@
         "symfony/security": "^4.3",
         "symfony/security-bundle": "^4.3",
         "symfony/swiftmailer-bundle": "^3.1.4",
+        "symfony/templating": "^4.3",
```

Comment:
Needed currently aslong https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2022 is not implemented